src/test/java/training/article/update.feature

```diff
@@ -0,0 +1,32 @@
+Feature: update an article
+
+  Background:
+    * url url
+    * def login = call read('../sign_in/sign_in.feature@loginSuccessfully')
+    * def username = username
+
+    * def token = "Bearer " + login.response.user.token
+    * def login = call read('create.feature@createSuccessfully')
+
+  Scenario Outline: update an article
+
+    * def updateArticle = {"article":{"title":'#(<updateTitle>)', "description":'#(<updateDescription>)', "body":'#(<updateDescriptionbody>)', "tagList":["dragons","training"]}}
+    * def articleSuccessfully = read('training/article/responsesuccessfully.json')
+
+    Given path 'articles', 'training-de-dllo-36xhmh'
```

Comment:
you don't have to harcode the slug, you must to get a random one and do the changes with that.